bootstrap/src/index.ts

```diff
@@ -42,9 +43,9 @@ const withStatusCode = (execute: ExecuteWrappedResponse) => async (data_: Adapte
 
 // Log adapter input & output data
 const withLogger = (execute: ExecuteWrappedResponse) => async (data: AdapterRequest) => {
-  logger.debug('Input: ', { input: data })
+  logger.debug({ input: data })
   const result = await execute(data)
-  logger.debug(`Output: [${result.statusCode}]: `, { output: result.data })
+  logger.debug({ output: result.data })
```

Comment:
Why remove the messages from this log outputs?

Reply:
```
{
  output: {
    jobRunID: '1',
    data: { result: '1316957652992' },
    result: '1316957652992',
    statusCode: 200
  },
  level: 'debug',
  message: 'Output: [200]: ',
  instanceId: '85070dc4-cdc7-4ac4-9269-5a9bdedee369',
  timestamp: '2021-01-12T19:02:36.199Z'
}
```

`output` can already been seen as the key to the data object. Doesn't seem to add any new useful information.

Reply:
It adds a `message`, you can think of it as a `label` that logs can be indexed by and then searched. How do we find this specific log otherwise?

We already have our metrics dependant on some of our log messages. For example while tracking caching hits/misses, we look for following logs by its `message/label`:

```javascript
logger.debug(`Cache: SET ${key}`, entry)
```

Think about how would we do that if we log just the `entry` here...

Reply:
Reverted this